packages/aws-cdk/lib/cli.ts

```diff
@@ -463,10 +476,16 @@ async function initCommandLine() {
           return cli.synth(args.STACKS, true, args.quiet, args.validation);
         }
 
-
       case 'metadata':
         return cli.metadata(args.STACK);
 
+      case 'acknowledge':
+        return cli.acknowledge(args.ID);
+
+      case 'notices':
+        SHOW_ALL_NOTICES = true;
```

Comment:
How about replacing this with:

```ts
DEFAULT_SHOW_NOTICES = false;
return showAllNotices({ /* config that ignores acks */ });
```

Does that not obviate the need of changing the return type the `initCommandLine` function? Plus, it has less "action at a distance" and reads more clearly (at least, to me).